### DISCUSSION TITLE
WM8960 Audio Codec

### DISCUSSION THREAD

**Discussion by syedadil19** [pschatzmann/arduino-audio-tools#1056] [Q&A] [answered]
Hi Phil, I was testing your code here https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-stream/streams-generator-wm8960/streams-generator-wm8960.ino on a WM8960 Audio Codec breakout board https://learn.sparkfun.com/tutorials/audio-codec-breakout---wm8960-hookup-guide/all. I used the connections you used here https://github.com/pschatzmann/arduino-wm8960/wiki/Pins-Connections. However, the example codes from SparkFun Library worked for me. But, I was hoping I could get your example to work as I could then integrate this Audio Codec as an external ADC for my current Audio BLE transmission project. I am currently using a ESP32-PICO-DevkitM-02 board with ESP32-PICO-MINI-02 chip. 

I've also tested your code with the WM8960 Audio Board which worked and produced a tone at my earphone. What's your thoughts on this?

**Comment by pschatzmann** [comment, marked as answer]
Just read the documentation from your codec carefully to figure out what additional pins you need to connect and handle.
Did you set the EN pin to high ?
